### PR TITLE
Update deployer kubectl versions

### PR DESCRIPTION
**Pull request by gibbleyg**

### CODE REVIEW

File: marketplace/deployer_helm_base/Dockerfile [hunk 1]
```diff
@@ -14,15 +14,15 @@ RUN pip3 install \
...
         && mkdir -p /opt/kubectl/$version \
         && wget -q -O /opt/kubectl/$version/kubectl \
             https://storage.googleapis.com/kubernetes-release/release/v$full_version/bin/linux/amd64/kubectl \
         && chmod 755 /opt/kubectl/$version/kubectl; \
      done;
-RUN ln -s /opt/kubectl/1.16 /opt/kubectl/default
+RUN ln -s /opt/kubectl/1.20 /opt/kubectl/default
 
 RUN mkdir -p /bin/helm-downloaded \
     && wget -q -O /bin/helm-downloaded/helm.tar.gz \
```
Comment (eshiroma) [inline marketplace/deployer_helm_base/Dockerfile:28]: Does it make sense to update the helm version as well?
(Not sure what changes have been made)

**Review by eshiroma — APPROVED** [review]
With kubectl supporting up to two minor versions earlier and 1.17 scheduled for end-of-life in Jan 2021 (already unavailable for new nodepools) as per the [GKE release schedule](https://cloud.google.com/kubernetes-engine/docs/release-schedule#schedule_for_static_no-channel_versions), this should be backwards-compatible enough with the clusters we expect to see in GCP for the apps that will be using this base deployer.

In the release, can we recommend that partners using these base deployers indicate a `clusterConstraints.k8sVersion` of at least `>=1.18` for this?

**Comment by gibbleyg** [comment]
> With kubectl supporting up to two minor versions earlier and 1.17 scheduled for end-of-life in Jan 2021 (already unavailable for new nodepools) as per the [GKE release schedule](https://cloud.google.com/kubernetes-engine/docs/release-schedule#schedule_for_static_no-channel_versions), this should be backwards-compatible enough with the clusters we expect to see in GCP for the apps that will be using this base deployer.
> 
> In the release, can we recommend that partners using these base deployers indicate a `clusterConstraints.k8sVersion` of at least `>=1.18` for this?

Will do!

**Pull request merged**